bucket/hourglass.json

```diff
@@ -0,0 +1,32 @@
+{
+    "homepage": "https://chris.dziemborowicz.com/apps/hourglass",
+    "description": "The simple countdown timer for Windows.",
+    "version": "1.10.0",
+    "license": "MIT",
+    "url": "https://github.com/dziemborowicz/hourglass/releases/download/v1.10.0/HourglassPortable.exe",
```

Comment:
Why you do not rename the file immediately on download rather then rename bin?

Reply:
How can I do to rename immediately? in `pre_install` script?

Reply:
```json
    "url": "https://github.com/dziemborowicz/hourglass/releases/download/v1.10.0/HourglassPortable.exe#/Hourglass.exe",
```

Reply:
Wa~~, Great!!